internal/config/instance.go

```diff
@@ -364,6 +353,10 @@ func (i *Instance) save() error {
 		return errs.Wrap(err, "Could not write config file")
 	}
 
+	if err = f.Sync(); err != nil {
```

Comment:
I added a file sync, as the lockfile really just synchronizes the time slice in which processes run these commands.  
As file operations (without sync) are usually deferred by the OS, it is possible that a file operation is actually executed (written to the hard-disk) while the lock is released already.  Reasonable OS implementations should ensure that there is still some sensible order of these writes, but I thought it wouldn't hurt not to rely on the OS here.